spec/javascripts/track-share-button-clicks.spec.js

```diff
@@ -9,18 +9,21 @@ describe('A share button click tracker', function () {
   var tracker
   var element
 
-  GOVUK.analytics = GOVUK.analytics || {trackShare: function () {}}
+  GOVUK.analytics = GOVUK.analytics || {}
+  GOVUK.analytics.trackShare = function () {}
 
   beforeEach(function () {
     tracker = new GOVUK.Modules.TrackShareButtonClicks()
   })
 
   it('tracks click events on share buttons', function () {
     spyOn(GOVUK.analytics, 'trackShare')
-    element = $('<div>\
-      <a href="#" class="js-share-facebook" data-network="facebook">Share</a>\
-      <a href="#" class="js-share-twitter" data-network="twitter">Share</a>\
-    </div>')
+    element = $(
+      '<div>' +
```

Comment:
Why was this updated?

I prefer writing long JS strings with `\` line-breaks, it's similar to the way we write Ruby arrays using `%w{}` – it requires less syntax.